vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -98,16 +98,21 @@ type internal GoToDefinition(checker: FSharpChecker, projectInfoManager: FSharpP
     member __.FindDefinitionOfSymbolAtRange(targetDocument: Document, symbolRange: range, targetSourceText: SourceText) =
         findSymbolHelper (targetDocument, symbolRange, targetSourceText, false)
     
-    /// use the targetSymbol to find the first instance of its presence in the provided source file
+    /// if the symbol is defined in the given file, ruturn it's declaration location, othewise use the targetSymbol to find the first 
```

Comment:
typo ruturn -> return.

Appreciates the comments BTW

Reply:
Thanks. Fixed.